modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaAddFolderParameterCheck.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.CharPool;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.source.formatter.check.util.JavaSourceUtil;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Nícolas Moura
+ */
+public class UpgradeJavaAddFolderParameterCheck extends BaseFileCheck {
+
+	@Override
+	protected String doProcess(
+			String fileName, String absolutePath, String content)
+		throws Exception {
+
+		Matcher matcher1 = _addFolderPattern.matcher(content);
+
+		String newContent = content;
+
+		while (matcher1.find()) {
+			String methodCall = JavaSourceUtil.getMethodCall(
+				content, matcher1.start());
+
+			List<String> parameterList = JavaSourceUtil.getParameterList(
+				methodCall);
+
+			if (parameterList.size() == 7) {
+				continue;
+			}
+
+			if (methodCall.contains("JournalFolderLocalServiceUtil")) {
+				newContent = _addParameter(newContent, methodCall);
+
+				continue;
+			}
+
+			String variable = methodCall.substring(
+				0, methodCall.indexOf(CharPool.PERIOD));
+
+			Pattern variableDeclarationPattern = Pattern.compile(
+				"[A-z]+ " + variable);
+
+			Matcher matcher2 = variableDeclarationPattern.matcher(content);
+
+			if (matcher2.find()) {
+				String declaration = matcher2.group();
```

Comment:
@nicolas-sss, can we rename the variable `declaration` to `variableDeclaration` instead in order to keep consistency?